internal/gengapic/gengapic.go

```diff
@@ -469,8 +469,7 @@ func (g *generator) comment(s string) {
 
 	lines := strings.Split(s, "\n")
 	for _, l := range lines {
-		l = strings.TrimSpace(l)
```

Comment:
This was removing the leading white space/indentation used to signify list items. There are many more places that trim white space before it is ultimately written, so trailing white space shouldn't be a concern.